test/e2e/serial/tests/resource_accounting.go

```diff
@@ -90,6 +90,10 @@ var _ = Describe("[serial][disruptive][scheduler][resacct] numaresources workloa
 	AfterEach(func() {
 		err := padder.Clean()
 		Expect(err).NotTo(HaveOccurred())
+
+		By("waiting for the NRT data to settle")
```

Comment:
is this (AfterEach()) the intended place? 
note that teardown is waiting for the NRT settlement to the initial data.

Reply:
right, that was the intent and I somehow fumbled. Fixing. Thanks for catching!

however I'm now also contemplating the actual utility about this change; we would have two waits back to back, whose value is questionable.